### PR TITLE
supporting PARENT_DICTIONARY key in dictionary file

### PR DESCRIPTION
**Pull request by vladimirgenestackcom**

### CODE REVIEW

**Comment by genestack-solomatin** [comment]
This branch has conflicts that must be resolved

**Comment by genestack-solomatin** [comment]
Please add link to related PRs

File: genestack_client/genestack_metainfo.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ class Metainfo(dict):
     CONTACT_PERSON = 'genestack:contactPerson'
     EXTERNAL_LINK = 'genestack:links'
     CREATION_DATE = 'genestack:dateCreated'
+    PARENT_DICTIONARY = 'genestack.dictionary:parent'
```
Comment (vasiliy-bout) [inline genestack_client/genestack_metainfo.py]: I'm afraid that this key must not be placed here... Need to consult with @genestack-solomatin though.

Comment (genestack-solomatin) [inline genestack_client/genestack_metainfo.py]: I try to move it and decide to leave it here.

We allow user to specify parent dictionary manually by passing metainfo with it. 

IMHO better place for this key in metainfo

**Pull request merged**